hack/kpng-local-up.sh

```diff
@@ -1,8 +1,10 @@
 #!/bin/bash
 # build the kpng image...
 
-: ${IMAGE:="jayunit100/kpng-server:latest"}
-
+: ${IMAGE:="jayunit100/kpng:ipvs"}
+echo $IMAGE
```

Comment:
change to -n `echo this will deploy kpng with docker image $IMAGE, pull policy $PULL and the $BACKEND backend. Press enter to confirm, C-c to cancel`